src/Output.ml

```diff
@@ -1,5 +1,5 @@
 let languageServerOutputChannel =
-  lazy (Vscode.Window.createOutputChannel "OCaml Language Server")
+  lazy (Vscode.Window.create_output_channel ~name:"OCaml Language Server")
```

Comment:
Why was this renamed? When creating bindings we should match original names whenever possible. Makes our bindings a lot easier to use.